tests/core.test.ts

```diff
@@ -0,0 +1,264 @@
+import { MPPAlert, Severity, alertType } from '../src/core/alert';
```

Comment:
In the tests, instead of checking for the status that the exception returns, check if the error is `instanceof` something.
For example, if the error should be `sqlRetryableError`, check if the error that you get is `instanceof sqlRetryableError` instead of checking that the status of the error is 1 or 2.